app/views/recipes/public_recipes.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= content_for :title, "Public Recipes" %>
+
+<h1>List of Public Recipes</h1>
+
+<% @recipes.each do |recipe| %>
+  <div class="recipe">
+    <h2><%= link_to recipe.name, recipe_path(recipe) %></h2> 
+    <p>Created by: <%= recipe.author.name %></p>
+   <ul class="public_list">
+    <li>Total Food Items: <%= recipe.foods.count %></li>
+    <li>Total Price: $<%= recipe.foods.sum(:price) %></li>
+</ul>
+  </div>
+<% end %>
```

Comment:
- [ ] It would be better to add margins on the edges of the following tables in order to match the design  and have clean layouts
- Yours 

<img width="1440" alt="Screenshot 2023-06-02 at 12 07 04" src="https://github.com/djo1975/Recipe-app/assets/61790587/def9d16b-5def-47da-825a-f6947380db88">
<img width="1440" alt="Screenshot 2023-06-02 at 12 07 11" src="https://github.com/djo1975/Recipe-app/assets/61790587/12468097-d426-4ba4-8719-cc5a4bd23c9f">
<img width="1440" alt="Screenshot 2023-06-02 at 12 07 44" src="https://github.com/djo1975/Recipe-app/assets/61790587/d8c79595-f533-42c3-956c-3004362d1298">

- Wireframe
<img width="722" alt="Screenshot 2023-06-02 at 12 10 30" src="https://github.com/djo1975/Recipe-app/assets/61790587/e556e53b-eec1-49fa-860c-033d08d1c0f5">